go.mod

```diff
@@ -86,10 +86,10 @@ require (
 	github.com/go-jose/go-jose/v3 v3.0.3
 	github.com/go-ldap/ldap/v3 v3.4.6
 	github.com/go-logr/logr v1.4.1
-	github.com/go-mysql-org/go-mysql v1.5.0 // replaced
+	github.com/go-mysql-org/go-mysql v1.7.0 // replaced
```

Comment:
Let's not touch this one, it's replaced. It's also in our ignore list.

https://github.com/gravitational/teleport/blob/master/.github/dependabot.yml#L9